frontend/docs/pages/sdks/python-sdk/asyncio.mdx

```diff
@@ -0,0 +1,112 @@
+# Working with AsyncIO
+
+Hatchet's Python SDK makes heavy use of AsyncIO under the hood to manage persistent connections,
+listeners, the runners for each workflow, and many other things! AsyncIO is a form of concurrency
```

Comment:
First thing to clarify might be that asyncio is optional -- perhaps something like "While using async methods is optional with Hatchet, using async methods incorrectly can cause issues and degrade the performance of the SDK."